conans/test/functional/configuration/profile_test.py

```diff
@@ -64,7 +64,7 @@ def test_profile_relative_cwd(self):
         self.client.save({"conanfile.txt": "", "sub/sub/profile": ""})
         self.client.current_folder = os.path.join(self.client.current_folder, "sub")
         self.client.run("install .. -pr=sub/profile2", assert_error=True)
-        self.assertIn("ERROR: Profile not found: sub/profile2", self.client.out)
+        self.assertIn("ERROR: Host profile: Profile not found: sub/profile2", self.client.out)
```

Comment:
Same as commented before. The message should be the same in the general usage without cross-build.

Reply:
We can add an `if` to check if the `build` profile has any value and remove the extra `Host profile:` from the error message ([conan_api.py#L1203](https://github.com/conan-io/conan/pull/5202/files#diff-633cc9158b73bec9defbb8eb390ba040R1203))